app/api/v2/views/store_views.py

```diff
@@ -82,49 +87,56 @@ def put(self,product_id):
         stock_amount = data["stock_amount"]
         price = data['price']
         low_inventory_stock = data['low_inventory_stock']
-        product=[product for product in products if product['product_id']==product_id]
+        product = [
+            product for product in products if product['product_id'] == product_id]
         if not product:
-            return make_response(jsonify({'Error':"Product Not found"}), 400)
-        new_pro=Products()
-        new_pro.update_product(product_id,product_name,category_id,stock_amount,price,low_inventory_stock)
-     
-        return make_response(jsonify({'Message':"{} Updated Successfuly".format(product[0]['product_name'])}), 200) #ok
-    
+            return make_response(jsonify({'Error': "Product Not found"}), 400)
+        new_pro = Products()
+        new_pro.update_product(
+            product_id, product_name, category_id, stock_amount, price, low_inventory_stock)
+
+        # ok
+        return make_response(jsonify({'Message': "{} Updated Successfuly".format(product[0]['product_name'])}), 200)
+
     @jwt_required
     @admin_required
-    def delete(self,product_id): 
+    def delete(self, product_id):
         """Delete product."""
         products = Products().get_all_products()
-        product=[product for product in products if product['product_id']==product_id]
+        product = [
+            product for product in products if product['product_id'] == product_id]
         if not product:
-            return make_response(jsonify({'Error':"Product Not found"}), 400)
-        new_pro=Products()
+            return make_response(jsonify({'Error': "Product Not found"}), 400)
+        new_pro = Products()
         new_pro.delete_product(product_id)
-        return make_response(jsonify({'Message':"Deleted Successfuly"}), 200) #ok
+        # ok
+        return make_response(jsonify({'Message': "Deleted Successfuly"}), 200)
+
 
 class ViewSalesRecord(Resource):
-    
+
     @jwt_required
     def get(self):
-        sales_record=Sales().get_all_sales()
+        sales_record = Sales().get_all_sales()
         """Get all sales' records."""
         if not sales_record:
-            return make_response(jsonify({"Message": "No Available sales records"}), 200)
+            return make_response(jsonify({"message": "No Available sales records"}), 200)
         return {"Sales Record": sales_record}, 200  # ok
 
     @jwt_required
     @admin_required
     def post(self):
         """ Make a new sale record."""
-        sales_record=Sales().get_all_sales()
+        sales_record = Sales().get_all_sales()
         products = Products().get_all_products()
         current_date = str(date.today())
         data = request.get_json(force=True)
-        #Validate().validate_empty_sales_inputs(data)
+        # Validate().validate_empty_sales_inputs(data)
```

Comment:
remove commented out methods that are not being used `#` this should be for comments and not functions you are not using.